tracing-futures/src/lib.rs

```diff
@@ -277,6 +277,16 @@ impl<T> Instrumented<T> {
         &mut self.span
     }
 
+    /// Borrows the wrapped type.
+    pub fn inner(&self) -> &T {
+        &self.inner
+    }
+
+    /// Mutably borrows the wrapped type.
+    pub fn inner_mut(&mut self) -> &mut T {
```

Comment:
When the `std-future` feature flag is enabled, the `Instrumented` type also has pin projections, we should probably expose those to the user as well eventually. I'm fine with doing that in a follow-up branch though.